.circleci/config.yml

```diff
@@ -12,44 +12,36 @@ commands:
   install_macos_dependencies:
     steps:
       - run: brew install protobuf cmake ccache libtool libspatialite pkg-config luajit curl wget czmq lz4 spatialite-tools unzip
-      - run: pip3 install requests shapely conan
+      - run: /usr/bin/python3 -m pip install --user requests
+      - run: pip3 install shapely conan
```

Comment:
`requests` is used by the `needs_ci_run` script which runs with `/usr/bin/python3`.

the other two libraries are used by cmake & valhalla and are run with `python3` which is apparently different from `/usr/bin/python3` so we need to install with that one as before